solr/core/src/test/org/apache/solr/cli/HealthcheckToolTest.java

```diff
@@ -61,7 +59,22 @@ public void testHealthcheckWithSolrUrlParameter() throws Exception {
     String solrUrl =
         ZkStateReader.from(cluster.getSolrClient()).getBaseUrlForNodeName(firstLiveNode);
 
-    String[] args = new String[] {"healthcheck", "-collection", "bob", "-solrUrl", solrUrl};
+    String[] args = new String[] {"healthcheck", "-c", "bob", "--solr-url", solrUrl};
+    assertEquals(0, runTool(args));
+  }
+
+  @Test
+  public void testHealthcheckWithSolrRootUrlParameter() throws Exception {
+
+    Set<String> liveNodes = cluster.getSolrClient().getClusterState().getLiveNodes();
+    String firstLiveNode = liveNodes.iterator().next();
+    String solrUrl =
+        ZkStateReader.from(cluster.getSolrClient()).getBaseUrlForNodeName(firstLiveNode);
+
+    // NOCOMMIT: epugh, why was this added?
+    // solrUrl = solrUrl.substring(0, solrUrl.indexOf("/solr"));
```

Comment:
@epugh I had to remove this for it to work, perhaps related to the recent #2519 ?

Reply:
Sorry, probably not related as that one only touches `bats` tests. Jason reverted the change in https://github.com/apache/solr/commit/b326f971f95c93216919cbfc2bf25c5d0853aa90

Reply:
I'm getting really confused about the `/solr` suffix. I add it some place to fix some tests, but then other tests break that assume the suffix is not supposed to be there...

Reply:
Finally nailed this!